src/App.css

```diff
@@ -23,10 +34,54 @@ h1 {
   font-family: 'Shantell Sans', cursive;
 }
 
+.Bookstore-CMS {
+  width: 15rem;
+  height: 2.313rem;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--azure);
+  margin: 2.313rem 6.25rem;
+}
+
+.BOOKS {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.CATEGORIES {
+  width: 6.563rem;
+  height: 1rem;
+  margin: 1.125rem 41.5rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
```

Comment:
In this activity is styling, we want the design of your app to closely mimic the [design template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) on Zeplin. Please bear in mind that the CSS code for this design can be easily obtained by simply clicking on a particular element of field and the CSS will be displayed on the right side, as seen in the attached screenshot below. Let's ensure to introduce the following improvements to our design:
- [ ] Ensure that all input fields, together with the add book button within the form area are on one horizontal line, as seen in the Zeplin template.
- [ ] Let's improve upon the typography by using the correct font face, size, and color
- [ ] Ensure to add the user icon at the top right
- [ ] Apply correct background styling
- [ ] Elements should be positioned correctly, and appropriate spacing added as seen in the Zeplin template.

**Your design:**
![image](https://github.com/mpa-mxiang/bookstores/assets/12745474/b67c3381-a59a-4749-92db-f17ab412a42e)

**Expected (You can find the CSS for a particular object and CSS properties by clicking on that object in Zeplin):**
![image](https://github.com/mpa-mxiang/bookstores/assets/12745474/bf885ecb-fb15-4b54-b256-899f16e18211)